src/encord_active/lib/project/project_file_structure.py

```diff
@@ -59,27 +59,42 @@ def get_encord_project_cached(ssh_key_path: str, project_hash: str) -> encord.Pr
 
 
 class LabelRowStructure:
-    def __init__(self, mappings: dict[str, str], label_hash: str, project: "ProjectFileStructure"):
+    def __init__(
+        self,
+        mappings: dict[str, str],
+        label_hash: str,
+        project: "ProjectFileStructure",
+        label_row: Optional["prisma.models.LabelRow"],
+    ):
         self._mappings: dict[str, str] = mappings
         self._rev_mappings: dict[str, str] = {v: k for k, v in mappings.items()}
         self._label_hash = label_hash
         self._project = project
+        self._label_row = label_row
 
     def __hash__(self) -> int:
         return hash(self._label_hash)
 
     def label_row_file_deprecated_for_migration(self) -> Path:
         return self._project.project_dir / "data" / self._label_hash / "label_row.json"
 
-    def get_label_row_json(self, cache_db: Optional[prisma.Prisma] = None) -> Dict[str, Any]:
+    def get_label_row_from_db(self, cache_db: Optional[prisma.Prisma] = None) -> "prisma.models.LabelRow":
```

Comment:
[nitpick] 💅

We could probably make this a `@property`

Reply:
I'll add a property variant for when cache_db is not needed.